### DISCUSSION TITLE
Simple Rollup Failure?

### DISCUSSION THREAD

**Discussion by MAEDSFDC** [SFDO-Community/declarative-lookup-rollup-summaries#1280] [General]
We have used the DLRS tool for many years, however I cam to create a really simple count of Opps through a custom lookup field and it didn't work. I then changed it to the Account and removed all the criteria, but still no joy? Triggers are deployed... just can't think why it wouldn't be working? I can't think of a much simpler scenario?

![image](https://user-images.githubusercontent.com/118757260/203109100-4b602253-229f-4d2c-b738-109747504554.png)

**Comment by aheber** [comment]
@MAEDSFDC the project directs support requests to our Trailblazer community which is much more heavily monitored and better supported by community members.
[Trailblazer DLSR Community Group](https://success.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F9300000009O5p)

When you head over there to request help you might find more success by including details of what isn't behaving correctly and including any errors you may have received.

If the problem is that it should be running but isn't I'd recommend trying to use the Full Caclulate button as a first attempt and see if that gets the data right, then you can narrow down the problem to the triggers/change reaction systems.

**Reply by MAEDSFDC** [reply]
Thanks for commenting. The full calculate does nothing, as does when I manually create an Opportunity. No errors, no rollups.
